# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../transcode/ffmpeg-version.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

===================================================================
RCS file: /cvstc/transcode/configure.in,v
retrieving revision 1.181
retrieving revision 1.183
diff -u -r1.181 -r1.183
--- transcode/configure.in	2006/06/07 00:52:26	1.181
+++ transcode/configure.in	2006/07/06 09:23:49	1.183
@@ -558,7 +558,13 @@
     printf("install ffmpeg 0.4.9-pre1 or newer, or a cvs version after 20040703");
     return(1);
   }
+
+  /* 3344640 == ((51<<16)+(9<<8)+0) == 51.9.0 */
+#if LIBAVCODEC_BUILD >= 3344640
+  printf("VER=%s\n", AV_STRINGIFY(LIBAVCODEC_VERSION));
+#else
   printf("VER=%s\n", FFMPEG_VERSION);
+#endif /* LIBAVCODEC_BUILD >= 51.9.0 check */
   printf("BUILD=%d\n", LIBAVCODEC_BUILD);
   return(0);
 }
